numpy/fft/_pocketfft_umath.cpp

```diff
@@ -183,12 +183,15 @@ rfft_impl(char **args, npy_intp const *dimensions, npy_intp const *steps,
          * Pocketfft uses FFTpack order, R0,R1,I1,...Rn-1,In-1,Rn[,In] (last
          * for npts odd only). To make unpacking easy, we place the real data
          * offset by one in the buffer, so that we just have to move R0 and
-         * create I0=0. Note that copy_data will zero the In component for
+         * create I0=0. Note that copy_input will zero the In component for
```

Comment:
This is weird - so, the statement I made in the comment is not true? I'll try to have a quick look, but in any case the fix clearly works.